grpc-server-runtime/build.gradle

```diff
@@ -1,4 +1,5 @@
 plugins {
+    id 'io.micronaut.build.internal.grpc-module'
```

Comment:
Would it make sense to have a convention plugin which applies both the `io.micronaut.build.internal.grpc-module` and `com.google.protobuf` plugins?

Reply:
Not sure because not all modules need the protobuf plugin. I think we can live with this and then we can improve it if needed in the future.